packages/shared/src/modules/users/libs/enums/user-validation-message.enum.ts

```diff
@@ -12,6 +12,7 @@ const UserValidationMessage = {
 		UserValidationRule.PASSWORD_MIN_LENGTH,
 	)} characters long`,
 	PASSWORD_REQUIRE: "Password is not allowed to be empty",
+	PASSWORD_RESET_MISMATCH: "Passwords do not match.",
```

Comment:
Why do you need to add this message here when the ConfirmPasswordCustomValidation enum already holds the same message?